library/ReplicatedTable.cs

```diff
@@ -2440,9 +2442,15 @@ private void MergeOperations(TableBatchOperation batch1, TableBatchOperation bat
 
         private void RunRepairInternal(RepairRowDelegate filter, DynamicReplicatedTableEntity entry, ref ReconfigurationStatus status)
```

Comment:
Please rename this method to RunRepairWithFilter

Reply:
Renamed to RepairRowWithFilter